### DISCUSSION TITLE
Will manually entered transactions match with imported bank transactions?

### DISCUSSION THREAD

**Discussion by phirestalker** [firefly-iii/firefly-iii#7314] [Firefly III] [answered]
I tried searching, but I couldn't seem to get the terms right, or nobody asked this before. Sorry if I am mistaken (feel free to point me to the existing answer).

I used to use Quicken a long time ago. The thing I liked most (because banks were much slower then) was entering a transaction manually, and days later when I downloaded the bank transactions it matched with my manual transactions. This is still useful today for some gas stations that stay at $1 for a day or two.

Can Firefly III do this? Will it recognize that I have already entered the transaction and match the imported bank one somehow?

**Comment by JC5** [comment, marked as answer]
Short answer: it won't.

**Comment by phirestalker** [comment]
Can I merge transactions manually?

**Reply by JC5** [reply]
Not yet, like I said this is on my list.

**Reply by phirestalker** [reply]
This is what happens when I'm lazy and reply by email. GitHub makes a new answer instead of a comment.
